Formula/s/scikit-image.rb

```diff
@@ -0,0 +1,86 @@
+class ScikitImage < Formula
+  include Language::Python::Virtualenv
+
+  desc "Image processing in Python"
+  homepage "https://scikit-image.org"
+  url "https://files.pythonhosted.org/packages/65/c1/a49da20845f0f0e1afbb1c2586d406dc0acb84c26ae293bad6d7e7f718bc/scikit_image-0.22.0.tar.gz"
+  sha256 "018d734df1d2da2719087d15f679d19285fce97cd37695103deadfaef2873236"
+  license "BSD-3-Clause"
+  head "https://github.com/scikit-image/scikit-image.git", branch: "main"
+
+  depends_on "libcython" => :build
+  depends_on "meson" => :build
+  depends_on "meson-python" => :build
+  depends_on "ninja" => :build
+  depends_on "pkg-config" => :build
+  depends_on "pythran" => :build
+  depends_on "numpy"
+  depends_on "pillow"
+  depends_on "python-packaging"
+  depends_on "python@3.11"
```

Comment:
it does not support py3.12?

Reply:
It depends on `scipy`, which is still on Python 3.11.